week24/day02/index.js

```diff
@@ -0,0 +1,12 @@
+const bobTestHelper = str => {
+  str = str.split(" ");
+  let hash = { odd: [], even: [] };
```

Comment:
a better way will be to keep summing elements and whenever it turns from odd to even, return the last element